app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,12 @@
+<footer class="footer">
+  <small>
+    The <%= link_to t("static_pages.link_title"), "http://www.railstutorial.org/" %> by <%= link_to "Michael Hartl", "http://www.michaelhartl.com/" %>
```

Comment:
em nên tìm hiểu cách truyền param vào i18n nhé. Chứ như này thì "The", "by" ... sẽ ko hiển thị theo đa ngôn ngữ được. 
Ý tưởng là mình sẽ chỉ thiết lập một biến i18n cho cả nguyên 1 câu (1 đoạn văn đầy đủ nghĩa), nhưng sẽ truyền thêm tham số là thẻ link_to. Chú ý các trường hợp tương tự